DESCRIPTION

```diff
@@ -0,0 +1,24 @@
+Package: dessert
+Type: Package
+Title: Follow your favourite analysis recipe with an automated dessert
```

Comment:
The title does not yet conform to the R packages advice: [https://r-pkgs.org/Metadata.html#description-title-description](https://r-pkgs.org/Metadata.html#description-title-description)

Reply:
have updated the title. Let's fill the text more in the coming weeks.